Model/pom.xml

```diff
@@ -166,6 +166,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>

```

Comment:
Since we're manually adding the postgres driver into tomcat, can you remove this?